### PR TITLE
OpenStack Swift configuration changed

### PR DESCRIPTION
**Pull request by mertcb**
OpenStack Swift configuration changed due to breaking change in @backstage/techdocs-common

Signed-off-by: Mert Can Bilgiç <mert.bilgic@trendyol.com>

### CODE REVIEW

**Comment by mertcb** [comment]
@emmaindal can you review this?

File: .changeset/odd-wasps-perform.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@techdocs/cli': minor
+---
+
+OpenStack Swift configuration changed due to OSS SDK Client change in @backstage/techdocs-common, it was a breaking change.
```
Comment (emmaindal) [inline .changeset/odd-wasps-perform.md:5]: Maybe add a reference to the PR https://github.com/backstage/backstage/pull/6839?

**Review by emmaindal — APPROVED** [review]
Awesome, thanks for updating this @mertcb Just one suggestion of referencing the PR with the techdocs-common changes, but otherwise this looks good!

**Comment by emmaindal** [comment]
Also some tests needs to be updated here I think https://github.com/backstage/techdocs-cli/blob/main/packages/techdocs-cli/src/lib/PublisherConfig.test.ts#L81

**Review by emmaindal — CHANGES_REQUESTED** [review]
As soon as tests are updated its a ✅ from me!

File: .changeset/odd-wasps-perform.md (hunk 1 above)
Comment (mertcb) [reply .changeset/odd-wasps-perform.md:5]: done that :)

**Comment by mertcb** [comment]
> As soon as tests are updated its a ✅ from me!

done:)

**Review by emmaindal — APPROVED** [review]

**Pull request merged**